spec/venue_spec.rb

```diff
@@ -32,24 +32,66 @@
 
   describe '#add_patron' do
     it 'returns a list of patrons' do
-      skip
+      # skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
-
       expect(venue.patrons).to eq ['Mike', 'Megan', 'Bob']
     end
   end
 
   describe '#yell_at_patrons' do
     it 'returns a list of uppercased names' do
-      skip
+      # skip
       venue = Venue.new('Bluebird', 4)
       venue.add_patron('Mike')
       venue.add_patron('Megan')
       venue.add_patron('Bob')
+      # require 'pry'; binding.pry
       expect(venue.yell_at_patrons).to eq ['MIKE', 'MEGAN', 'BOB']
     end
   end
+
+  #Iteration 3
+
+  describe 'over_capacity' do
+    it 'it is under capacity' do
```

Comment:
You can just use your next test and get rid of this one. The next test is testing for both `true` and `false`. Also make sure to remove your commented out `pry`